console/module/api/src/main/resources/org/eclipse/kapua/app/console/module/api/client/messages/ValidationMessages.properties

```diff
@@ -55,7 +55,7 @@ invalidNullValue=This value cannot be null.
 lastAdministrator=Cannot delete the last Administrator of an Account or remove its Administrator Role.
 
 # Exception Messages
-UNAUTHENTICATED=You are not logged in.
+UNAUTHENTICATED=You are not logged in due to 30 min of inactivity!
```

Comment:
The session inactivity is not strictly 30 minutes, since it's configurable via web.xml. So, although uncommon, this could lead to an incorrect message. I'd rather leave this line as it is now without adding the 30 minutes delay.

Reply:
@lorthirk error message reverted.